classes/03class/exercises/c03-docker01/christina-roperto/c03-docker01.txt

```diff
@@ -0,0 +1,34 @@
+Command: $ docker build -t da-terrrafrom:c03-docker-e01 . && docker run --rm -it da-terrrafrom:c03-docker-e01
+
+To build: $ docker build -t da-terrrafrom:c03-docker-e01 . 
+
+Sending build context to Docker daemon  2.048kB
+Step 1/5 : from alpine
+ ---> e7d92cdc71fe
+Step 2/5 : RUN wget https://releases.hashicorp.com/terraform/0.12.23/terraform_0.12.23_linux_amd64.zip
+ ---> Using cache
+ ---> eeb2877ec45a
+Step 3/5 : RUN unzip terraform_0.12.23_linux_amd64.zip
+ ---> Using cache
+ ---> dc116cadd7a9
+Step 4/5 : RUN rm terraform_0.12.23_linux_amd64.zip
+ ---> Running in 9e07b04445ed
+Removing intermediate container 9e07b04445ed
+ ---> c56769096143
+Step 5/5 : CMD ./terraform --version
+ ---> Running in 4470b16bb08d
+Removing intermediate container 4470b16bb08d
+ ---> 89460800358f
+Successfully built 89460800358f
+Successfully tagged da-terrrafrom:c03-docker-e01
+
+To execute: $ docker run --rm -it da-terrrafrom:c03-docker-e01
+Output:
+Terraform v0.12.23
```

Comment:
Can you change the terraform cli version to version 0.12.20, as indicated in the exercise?

Reply:
Sure, no worries. Just curious though, is there any reason why we shouldn't use the latest version?

Reply:
Nothing specific..